pkg/master/master.go

```diff
@@ -586,12 +593,12 @@ func (m *Master) initV1ResourcesStorage(c *Config) {
 }
 
 // NewBootstrapController returns a controller for watching the core capabilities of the master.
```

Comment:
Comment whether endpointReconciler may be nil-- you pass nil in some tests and I wouldn't expect that to be valid.